src/semantic-ui/site/collections/breadcrumb.overrides

```diff
@@ -0,0 +1,3 @@
+/*******************************
+         Site Overrides
```

Comment:
Assuming these are Semantic-UI artifacts? Are they used to override styles of the built-in components? Sure are a lot of them... Is there a way to suppress them if they're not needed?